frontend/public/extend/devconsole/components/ImportFlowForm/ImportFlowForm.tsx

```diff
@@ -378,12 +516,23 @@ export class ImportFlowForm extends React.Component<Props, State> {
           this.setState({
             gitUrlValidationStatus: res.status.connection.state,
             gitRepoUrlError: '',
+            showSourceSecretDropDown: false,
           });
         } else {
-          this.setState({
-            gitUrlValidationStatus: res.status.connection.state,
-            gitRepoUrlError: getUrlErrorMessage(res.status.connection.reason),
-          });
+          if (res.status.connection.reason === 'RepoNotReachable') {
```

Comment:
Use enum here instead of string literals. The whole reason of having enums is to use constants defined only in one place and used elsewhere.
```suggestion
          if (res.status.connection.reason === ErrorMessage.RepoNotReachable) {
```